R/toml_serialization.R

```diff
@@ -84,18 +83,18 @@ make_header <- function(nm, arg) {
 
 make_idx_msg <- function(invalid, args_limit = 5L) {
   idx <- paste0( # nolint: object_usage_linter
-    glue("`{utils::head(invalid, args_limit)}`"),
+    glue::glue("`{utils::head(invalid, args_limit)}`"),
     collapse = ", "
   )
   if (length(invalid) > args_limit) {
-    idx <- glue("{idx}, ... ")
+    idx <- glue::glue("{idx}, ... ")
   }
```

Comment:
This is confusing code, but it wasn't introduced in this PR so is probably out of scope.

Reply:
This is sketchy toml serialization logic we invented some time ago :-D